src/wp-content/themes/twentytwenty/style.css

```diff
@@ -5359,6 +5359,11 @@ a.to-the-top > * {
 		font-size: 2.8rem;
 	}
 
+	.wp-block-quote.has-text-color cite,
```

Comment:
In style.css, these rules only apply at 782 pixels and wider. The style-rtl.css version has the rules earlier (and not indented).

Reply:
Do you recommend to remove it from media query and add it in the global css (without any media query)?